### PR TITLE
Added functionality to show errors in frontend on faulty filetypes

### PR DESCRIPTION
**Pull request by jkorpelin**
Added frontend functionality to show faulty filetypes and separated stuff from commands to document model cljc.

### CODE REVIEW

File: app/frontend/src/cljs/teet/ui/file_upload.cljs [hunk 1]
```diff
@@ -148,8 +151,17 @@
...
+         (.log js/console file)
+         (let [invalidfile? (not (s/valid? :document/file-object file))
+               file (document-model/file-info file)]
+           [ListItemText (merge
+                           {:primary (:file/name file)
+                            :secondary (r/as-component [:<>
+                                                        [:span (merge {} (when invalidfile?
+                                                                           {:style {:color theme-colors/error}}))
+                                                         (str (:file/type file)) (when invalidfile?
+                                                                                   (str " " (tr [:document :wrong-file-type])))]
```
Comment (tatut) [inline app/frontend/src/cljs/teet/ui/file_upload.cljs]: other causes of invalidity may be misreported as "wrong file type" here, especially if we add size checks later

File: app/frontend/src/cljs/teet/ui/file_upload.cljs [hunk 2]
```diff
@@ -148,8 +151,17 @@
       (fn [i ^js/File file]
         ^{:key i}
         [ListItem {}
-         [ListItemText {:primary (.-name file)
-                        :secondary (str (.-type file) " " (format/file-size (.-size file)))}]
+         (.log js/console file)
```
Comment (erno) [inline app/frontend/src/cljs/teet/ui/file_upload.cljs]: left over .log

File: app/common/src/cljc/teet/document/document_model.cljc [hunk 3]
```diff
@@ -0,0 +1,38 @@
+(ns teet.document.document-model
+  (:require [clojure.string :as str]))
+
+(def ^:const upload-max-file-size (* 1024 1024 100))
```
Comment (erno) [inline app/common/src/cljc/teet/document/document_model.cljc]: 100 MB seems to me a small limit for some kinds of files (detailed technical PDFs, zip files) ?

File: app/frontend/src/cljs/teet/ui/file_upload.cljs (hunk 1 above)
Comment (jkorpelin) [reply app/frontend/src/cljs/teet/ui/file_upload.cljs]: Changed the check to only check for filetype now. And if we add check for size need to add different error style.

File: app/frontend/src/cljs/teet/document/document_controller.cljs [hunk 4]
```diff
@@ -142,6 +140,7 @@
     (let [doc-id (goog.math.Long/fromString (get-in app [:query :document]))]
       (t/fx app
             (fn [e!]
+              (println "FILESSSS: " files)
```
Comment (outamaa) [inline app/frontend/src/cljs/teet/document/document_controller.cljs]: Mur.

**Pull request merged**